charts/descheduler/templates/clusterrole.yaml

```
@@ -24,6 +24,12 @@ rules:
 - apiGroups: ["scheduling.k8s.io"]
   resources: ["priorityclasses"]
   verbs: ["get", "watch", "list"]
+- apiGroups: [""]
+  resources: ["persistentvolumeclaims"]
+  verbs: ["get", "watch", "list", "delete"]
+- apiGroups: [""]
+  resources: ["persistentvolumes"]
+  verbs: ["get", "watch", "list"]
```

Should these be guarded by the policy's condition?

```suggestion
{{- if not (default true | deschedulerPolicy.IgnoreLocalPVCPods) }}
- apiGroups: [""]
  resources: ["persistentvolumeclaims"]
  verbs: ["get", "watch", "list", "delete"]
- apiGroups: [""]
  resources: ["persistentvolumes"]
  verbs: ["get", "watch", "list"]
{{ end }}
```

---

Good idea, if `ignoreLocalPvcPods` is `true` then there is no reason to have `delete` ability for PVCs